.github/CODEOWNERS

```diff
@@ -142,29 +150,36 @@
 # New structures team.
 # Question project.
 /core/controllers/question*.py @aks681
-/core/domain/question*.py  @aks681
+/core/domain/question*.py @aks681
 /core/storage/question/ @aks681
 /core/templates/dev/head/domain/question/ @aks681
 /core/templates/dev/head/pages/question_editor/ @aks681
 /core/templates/dev/head/pages/questions_list/ @aks681
 
+# Practice session project.
+/core/templates/dev/head/pages/practice_session/ @aks681
+/core/controllers/practice_sessions*.py @aks681
+
 # Skill project.
+/core/controllers/concept_card_viewer*.py @aks681
```

Comment:
@aks681 I'm not sure where to place this (As this is also used in the exploration) so can you confirm that we can place this over here?

Reply:
Yeah, it's fine to place it here.